content/sensu-core/2.0/reference/backend.md

```diff
@@ -0,0 +1,585 @@
+      --etcd-listen-client-urls string            list of URLs to listen on for client traffic (default "http://127.0.0.1:2379")
+      --etcd-listen-peer-urls string              list of URLs to listen on for peer traffic (default "http://127.0.0.1:2380")
+      --etcd-name string                          human-readable name for this member (default "default")
+      --etcd-peer-cert-file string                path to the peer server TLS cert file
+      --etcd-peer-client-cert-auth                enable peer client cert authentication
+      --etcd-peer-key-file string                 path to the peer server TLS key file
+      --etcd-peer-trusted-ca-file string          path to the peer server TLS trusted CA file
+      --etcd-trusted-ca-file string               path to the client server TLS trusted CA cert file
+      --no-embed-etcd                             don't embed etcd, use external etcd instead
+{{< /highlight >}}
+
+### General configuration flags
+
+| config-file |      |
+--------------|------
+description   | Path to Sensu backend config file
+type          | String
+default       | <ul><li>Windows: `C:\\ProgramData\sensu\config\backend.yml`</li><li>FreeBSD: `/usr/local/etc/sensu/backend.yml`</li><li>Linux: `/etc/sensu/backend.yml`</li></ul>
+example       | {{< highlight shell >}}# Command line example
+sensu-backend start --config-file /etc/sensu/backend.yml
+sensu-backend start -c /etc/sensu/backend.yml
+
+# /etc/sensu/backend.yml example
+config-file: "/etc/sensu/backend.yml"{{< /highlight >}}
+
+
+| debug     |      |
+------------|------
+description | Enable debugging and profiling features
+type        | Boolean
+default     | `false`
+example     | {{< highlight shell >}}# Command line example
+sensu-backend start --debug
+
+# /etc/sensu/backend.yml example
+debug: true{{< /highlight >}}
+
+
+| deregistration-handler |      |
+-------------------------|------
+description              | Default deregistration handler
+type                     | String
+default                  | `""`
+example                  | {{< highlight shell >}}# Command line example
+sensu-backend start --deregistration-handler /path/to/handler.sh
+
+# /etc/sensu/backend.yml example
+deregistration-handler: "/path/to/handler.sh"{{< /highlight >}}
+
+
+| log-level  |      |
+-------------|------
+description  | Logging level: `panic`, `fatal`, `error`, `warn`, `info`, or `debug`
+type         | String
+default      | `"warn"`
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --log-level debug
+
+# /etc/sensu/backend.yml example
+log-level: "debug"{{< /highlight >}}
+
+
+| state-dir  |      |
+-------------|------
+description  | Path to Sensu state storage 
+type         | String
+default      | <ul><li>Windows: `C:\\ProgramData\sensu\data`</li><li>FreeBSD: `/usr/local/etc/sensu/backend.yml`</li><li>Linux: `/var/lib/sensu`</li></ul>
```

Comment:
For Windows and Linux, these refer to directories. For FreeBSD, it is a file. Should it be `/usr/local/etc/sensu` as this section is for `state-dir`?